src/sections/stats/sections/LRNA/components/Burning/Burning.tsx

```diff
@@ -0,0 +1,94 @@
+import { PieChart } from "../PieChart/PieChart"
+import { ReactComponent as StakingRewardIcon } from "assets/icons/StakingRewardIcon.svg"
+import { ReactComponent as StakedAPRIcon } from "assets/icons/StakedAPR.svg"
+import { Icon } from "components/Icon/Icon"
+import { Text } from "components/Typography/Text/Text"
+import { SBurnContainer } from "./Burning.styled"
+import { useTranslation } from "react-i18next"
+import { useLRNAMeta } from "api/assetMeta"
+import { useHubAssetImbalance } from "api/omnipool"
+import { useApiPromise } from "utils/api"
+import { formatValue } from "../../StatsLRNA.utils"
+import BigNumber from "bignumber.js"
+import { BN_0 } from "utils/constants"
+import { useDisplayAssetStore } from "utils/displayAsset"
+import { useSpotPrice } from "api/spotPrice"
+
+export const Burning = () => {
+  const api = useApiPromise()
+  const { t } = useTranslation()
+
+  const meta = useLRNAMeta(api)
```

Comment:
It's better to use useApiPromise inside the useLRNAMeta hook, so it will be same across all hooks